src/main/java/complete_task/completeTaskController.java

```diff
@@ -1,20 +1,18 @@
 package complete_task;
-import entity_layer.Curriculum;
 
 /**
  * Controller for the completeTask use case; receives user input in
- * the form of the id of the task to be completed and the curriculum
- * that the task belongs to.
+ * the form of the id of the task to be completed, the id of the
+ * curriculum that the task belongs to and the active user.
  *
  * @author Bmguiler
  */
 public class completeTaskController {
     public static int taskId;
-    public static Curriculum taskCurriculum;
+    public static int taskCurriculumId;
 
-    public completeTaskController(int taskId, Curriculum taskCurriculum){
+    public completeTaskController(int taskId, int taskCurriculumId){
```

Comment:
It would be best instead of storing these as instance attributes, passing them to an interactor

Reply:
David has a point. If you look in the example from lecture, the only thing the Controller is initialized with is with a reference to the UserInputBoundary it will depend on. Then, it will have a functional method that takes information from the UI (user input) and passes it to said InputBoundary (a UseCaseInteractor at runtime) for it to be processed.